src/backend/distributed/sql/udfs/citus_prepare_pg_upgrade/latest.sql

```diff
@@ -67,6 +69,22 @@ BEGIN
        objects.colocationid
     FROM citus.pg_dist_object objects,
          pg_catalog.pg_identify_object_as_address(objects.classid, objects.objid, objects.objsubid) address;
+
+    SELECT count(*) > 0
```

Comment:
minor: needs comment? What are we doing here? Why are we dropping citus_stat_activity?